src/vcState.h

```diff
@@ -115,10 +122,12 @@ struct vcState
   bool forceLogout;
   double lastServerAttempt;
   double lastServerResponse;
-  vdkContext *pVDKContext;
-  vcRenderContext *pRenderContext;
+  vcContextIndex eCurrentContextIndex;
+  vdkContext *pVDKContext[vcContext_Count];
```

Comment:
There should only be 1 pVDKContext; This will use multiple licenses for no reason?